avm/res/container-service/managed-cluster/main.bicep

```diff
@@ -304,6 +304,12 @@ param enableWorkloadIdentity bool = false
 @description('Optional. Whether to enable Azure Defender.')
 param enableAzureDefender bool = false
 
+@description('Optional. Whether to enable Image Cleaner for Kubernetes.')
+param enableImageCleaner bool = false
+
+@description('Optional. The interval in hours Image Cleaner will run. Minimum value is 24 hours and maximum is three months.')
+param imageCleanerIntervalHours int = 24
```

Comment:
```suggestion
@description('Optional. The interval in hours Image Cleaner will run. The maximum value is three months.')
@minValue(24)
param imageCleanerIntervalHours int = 24
```

Reply:
Woud have liked to add a maxValue as well, but `3 months` is a bit ... odd to calculate as not every month has the same length. I guess, `3 x 31 days x 24 hours = 2232` would at least set some limit, but I'm not sure if it's a helpful limit.

Reply:
Makes sense to me.